modules/server/src/main/ts/BedrockManual.ts

```diff
@@ -4,13 +4,15 @@ import * as RunnerRoutes from './bedrock/server/RunnerRoutes';
 import * as Webpack from './bedrock/compiler/Webpack';
 import { BedrockManualSettings } from './bedrock/core/Settings';
 import { ExitCodes } from './bedrock/util/ExitCodes';
+import * as SettingsResolver from './bedrock/core/SettingsResolver';
 
-export const go = (settings: BedrockManualSettings): void => {
+export const go = (bedrockManualSettings: BedrockManualSettings): void => {
+  console.log('bedrock-manual ' + Version.get() + ' starting...');
+
+  const settings = SettingsResolver.resolveAndLog(bedrockManualSettings);
```

Comment:
Is there a reason you put the blank line above this, instead of below like in `BedrockAuto.ts`?

Reply:
No reason. Fixed.